bundle/OpenGraph/Handler/ImageFallback.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\OpenGraph\Handler;
+
+use Netgen\Bundle\OpenGraphBundle\Handler\ContentAware;
+use eZ\Publish\Core\Base\Exceptions\InvalidArgumentException;
+use eZ\Publish\API\Repository\Values\Content\Field;
+use Netgen\Bundle\OpenGraphBundle\Handler\HandlerInterface;
+use Netgen\Bundle\OpenGraphBundle\Handler\FieldType\Handler;
+use eZ\Publish\Core\FieldType\Image\Value as ImageValue;
+use Netgen\Bundle\RemoteMediaBundle\Core\FieldType\RemoteMedia\Value as RemoteImageValue;
+use eZ\Publish\Core\Helper\FieldHelper;
+use eZ\Publish\Core\Helper\TranslationHelper;
+use Netgen\Bundle\OpenGraphBundle\Exception\FieldEmptyException;
+use Netgen\Bundle\OpenGraphBundle\Handler\FieldType\Image as ImageHandler;
+use Netgen\Bundle\RemoteMediaBundle\OpenGraph\Handler\RemoteMediaHandler;
+
+final class ImageFallback extends Handler implements HandlerInterface, ContentAware
+{
+    private $ezImageHandler;
```

Comment:
Missing property typehints.

Reply:
Done in a8739a6e0b8ef5b1cdfe47fc77b0e52235296df3